workflows/templates/cilium.migrate.yaml

```diff
@@ -99,7 +99,13 @@ spec:
               - name: scriptContent
                 value: |
                   echo "restarting daemonsets on {{inputs.parameters.targetNcn}}"
-                  /srv/cray/scripts/common/restart-ds.sh {{inputs.parameters.targetNcn}}
+                  NODE={{inputs.parameters.targetNcn}}
+                  kubectl get nodes --no-headers=true | awk '{print $1}' | grep -q "^${NODE}$" || { echo "ERROR: Invalid node name ${NODE}"; exit 1; }
```

Comment:
This line will work, but it could be simplified to:
```
kubectl get node ${NODE} >/dev/null 2>&1 || { echo "Error: Invalid node name ${NODE}; exit 1; }
```

Reply:
Fixed